Dockerfile

```diff
@@ -45,8 +46,8 @@ RUN echo "Will install xrootd version: $XROOTD_VERSION (latest if empty)" && \
     yum install -y xrootd-"$XROOTD_VERSION" python3-xrootd-"$XROOTD_VERSION" && \
     yum clean -y all
 
-RUN pip uninstall pipenv -y && pip install --no-cache-dir --upgrade pip==20.2.4 setuptools==68.2.2 wheel==0.36.2 && \
-    npm install -g --unsafe-perm node-sass@6.0.1 clean-css@3.4.24 requirejs@2.3.6 uglify-js@3.12.1 jsonlint@1.6.3 d3@6.3.1 \
+RUN pip uninstall pipenv -y && pip install --no-cache-dir --upgrade pip==24.3.1 setuptools==70.0.0 wheel==0.45.1 && \
```

Comment:
I like that we are going for newer versions. Are those the latest available? If so, would it make sense to change it to require `=>` instead of `==`?

Reply:
`pip` and `wheel` are the latest right now, I cannot set the version of `setuptools` to `71.0.0 or higher` because of other issues during the build (latest there is `75.6.0`)

Reply:
Also, I just fixed that, but the dockerfile-linter does not appreciate the command like this